packages/knip/test/plugins/yarn.test.ts

```diff
@@ -0,0 +1,23 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/yarn');
+
+test('Find dependencies with the yarn plugin', async () => {
+  const { issues, counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert(issues.files.size === 0);
```

Comment:
Just saying, `baseCounters` contains `files: 0` which is essentially the same assertion (no need to change though).